### DISCUSSION TITLE
Node in Nodes collection do not report a MIXED state, only IDLE or ALLOCATED

### DISCUSSION THREAD

**Discussion by robgics** [PySlurm/pyslurm#316] [General]
I used the collection to load all nodes, and then I print out the state of all of them.  They are all either IDLE, ALLOCATED, or some form of DOWN/DRAIN...but no MIXED.   Inspecting particular nodes that I know are mixed (via scontrol show node) shows that if a node is MIXED (so even 1 cpu in use, out of 48), then it will appear as ALLOCATED via pyslurm.

I *think* we can work around this by comparing total cpus and allocated cpus.  But I'm just wondering if this simplification is intentional or if I'm missing something.  

Thanks!

**Comment by tazend** [comment]
Same here, created a new issue #317, lets continue there.
